examples/model_selection/plot_nested_cross_validation_iris.py

```diff
@@ -90,7 +90,7 @@
 
 score_difference = non_nested_scores - nested_scores
 
-print("Average difference of {0:6f} with std. dev. of {1:6f}."
+print("Average difference of {:6f} with std. dev. of {:6f}."
```

Comment:
Is this intentional? Not that I actually care ;)

Reply:
I have not bothered reverting a few such changes, as long as there are no 500 lines of those, I'm hoping it wouldn't matter much.

Reply:
fine with me.